docs/index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   hpe-csi-driver:
+  - apiVersion: v1
+    appVersion: 1.1.1
+    created: "2020-04-16T15:07:13.117530849+05:30"
+    description: A Helm chart for installing the HPE CSI Driver for Kubernetes
+    digest: 770214da764d108d5261e82904c58674b7e496e00afe4a6b57616f763edc7428
+    home: https://hpe.com/storage/containers
+    icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
+    keywords:
+    - HPE
+    - Storage
+    - StorageClass
+    - CentOS
+    - Ubuntu
+    - RHEL
+    maintainers:
+    - email: hpe-containers-dev@hpe.com
+      name: shivamerla
+    name: hpe-csi-driver
+    sources:
+    - https://github.com/hpe-storage/csi-driver
+    urls:
+    - hpe-csi-driver-2.0.0.tgz
+    version: 2.0.0
```

Comment:
Can you change version to 1.2.0 and delete old tgz 2.0.0-beta.

Reply:
done